lib/modules/manager/gitlabci/extract.ts

```diff
@@ -122,9 +122,16 @@ function extractDepFromIncludeComponent(
   includeComponent: GitlabIncludeComponent,
   registryAliases?: Record<string, string>,
 ): PackageDependency | null {
-  const componentReference = componentReferenceRegex.exec(
-    includeComponent.component,
-  )?.groups;
+  let componentPath = includeComponent.component;
+  const componentFQDN = componentPath?.substring(0, componentPath.indexOf('/'));
```

Comment:
```suggestion
  const componentFQDN = componentPath.substring(0, componentPath.indexOf('/'));
```
I think component path can't be empty with your changed check above.